rules/style.js

```diff
@@ -14,9 +14,28 @@ module.exports = {
         'camelcase': 2,
         // Enforce or disallow capitalization of the first letter of a comment
         'capitalized-comments': [2, 'always', {
-            // Ignore comments that have possible code samples as the first word (camelCase, some-prop, fn(), etc.)
-            // Additionally, ignore `falls through` because of switch statements
-            'ignorePattern': '([a-z0-9]+[A-Z_\\-.()\'"`]|falls?\\s?through)',
+            'ignorePattern': [
+                // Ignore comments that have possible code samples as the first word (camelCase, some-prop, fn(), etc.)
+                '[a-z0-9]+[A-Z_\\-.()\'"`]',
+                // Ignore `falls through` because of switch statements
+                'falls?\\s?through',
+                // Ignore language reserved keywords to allow commenting code
+                'import',
```

Comment:
I think that instead of having a lot of these keywords, we could add simpler ones that would cover these and more.
For instance, `Object.keys(foo).forEach((key) => console.log('key', key);` would not be covered by language constructs.

If we analyze JavaScript, some statements end with a `;`, except a few constructors, such as `for`, `while` and `do while`. Though, those also have `{` and `}` in the same line. My suggestion is to look at some sample code [from](https://github.com/moxystudio/node-proper-lockfile) [our](https://github.com/moxystudio/node-cross-spawn) [projects](https://github.com/moxystudio/node-promptly) to see if both of these chars cover all the use cases.

What are your thoughts?

Reply:
Unfortunately, it looks like `capitalized-comments`'s `ignorePattern` only works for the first word 👎. Having that said, we have to go forward with your solution. Perhaps, we should add a few more keywords, such as `for`, `while` and `do`. 

Additionally, if the first word contains `(` or `{`, it also should be ignored, e.g.: `arr.forEach((key) => console.log('key', key);`. Just add `{` and `}` to the first regular expression and it should be good.

Reply:
Is there any reason to not include all the reserved [keywords](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar) ? 

I would sort the keywords list, just to be easier to check.

Reply:
@andregoncalvesdev indeed, we should add all of them. Neverthess, my suggestion to add `{` and `}` to the first pattern should solve `arr.forEach((key) => console.log('key', key);` and similar.

Reply:
Yes, unfortunately it only works for the first word. 😞 
I'll have a look at the grammar and other edge cases, sort and complete the list.

Thanks for the feedback! 👍

Reply:
@satazor the code block you posted above is not flagged by `capitalized-comments` already.